platform/ios/platform/ios/src/MGLMapProjection.mm

```diff
@@ -0,0 +1,121 @@
+//
+//  MGLMapProjection.m
+//  ios
+//
+//  Created by Oleksandr Stepanov on 07.12.2021.
```

Comment:
I leave it up to you if you want to include your name.

Reply:
I reviewed other .mm files, and removed this copyright header at all, to be aligned with the rest of codebase.